src/index.css

```diff
@@ -0,0 +1,262 @@
+    -moz-border-radius: 12px;
+    -ms-border-radius: 12px;
+    -o-border-radius: 12px;
+    border-radius: 12px;
+    background: #FFF;
+    box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.10);
+    -webkit-transform:translate(-50%,-50%);
+    -moz-transform:translate(-50%,-50%);
+    -ms-transform:translate(-50%,-50%);
+    -o-transform:translate(-50%,-50%);
+}
+.uploading p{
+align-self: flex-start;
+color:#4F4F4F;
+font-family: Poppins;
+font-size: 18px;
+font-style: normal;
+font-weight: 500;
+line-height: normal;
+letter-spacing: -0.63px;
+margin-bottom: 30.74px;
+}
+.loading{
+    position: relative;
+width: 340.711px;
+height: 6px;
+flex-shrink: 0;
+border-radius: 8px;
+background:  #F2F2F2;
+overflow: hidden;
+
+}
+
+@keyframes loader {
+    0% {
+      left: 0;
+      transform: translateX(10%);
+      -webkit-transform: translateX(10%);
+      -moz-transform: translateX(10%);
+      -ms-transform: translateX(10%);
+      -o-transform: translateX(10%);
+}
+    100% {
+      left: 80%;
+      transform: translateX(40%);
+      -webkit-transform: translateX(40%);
+      -moz-transform: translateX(40%);
+      -ms-transform: translateX(40%);
+      -o-transform: translateX(40%);
+}
+  }
+.blue{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100.929px;
+    height: 6px;
+    flex-shrink: 0;
+    border-radius: 8px;
+    background: #2F80ED;
+    animation: loader 1s linear infinite;
+    -webkit-animation: loader 1s linear infinite;
+}
+.success-container{
+    padding: 36px 31.18px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 400.359px;
+    height: 454.956px;
+    flex-shrink: 0;
+    border-radius: 12px;
+    background: #FFF;
+    box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.10);
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform:translate(-50%,-50%);
+}
+.success-container p{
+    color:  #4F4F4F;
+    text-align: center;
+    font-family: Poppins;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+    letter-spacing: -0.63px;
+
+}
+.div-image{
+    width: 338px;
+    height: 224.397px;
+    flex-shrink: 0;
+    border-radius: 12px;
+    margin-top: 25.34px;
+    margin-bottom: 25.34px;
+}
+.link{
+    width: 338px;
+    height: 34px;
+    flex-shrink: 0;
+    position: relative;
+
+}
+.link input {
+    width: 338px;
+    height: 34px;
+   border-radius: 8px;
+   border: 1px solid var(--gray-5, #E0E0E0);
+   background: #F6F8FB;
+   padding-left: 7.12px;
+}
+.link button{
+    color: #FFF;
+    text-align: center;
+    font-family: Poppins;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+    letter-spacing: -0.28px;
+    width: 74px;
+    height: 30px;
+    flex-shrink: 0;
+    border-radius: 8px;
+    background:  #2F80ED;
+    position: absolute;
+    outline: none;
+    border: none;
+    right: 2px;
+    top: 2px;
+    bottom: 2px;
+}
```

Comment:
Please format this file, you can use a VS Code plugin called prettier, this will auto-format the code on save.

Reply:
Solved, thanks for your review and your advice.